utapiServer.js

```diff
@@ -1,4 +0,0 @@
-'use strict'; // eslint-disable-line strict
```

Comment:
I will just double-check if @rahulreddy has an opinion on this since he was thinking we can remove the unnecessary files now. Maybe move the lib/utapi files to the root instead?